src/remoteeventhandler.ts

```diff
@@ -271,10 +271,13 @@ export class RemoteEventHandler {
 						const bodyParts = info.message.body.split("\n");
 						bodyParts[0] = `${info.message.emote ? "* " : ""}<${info.user.mxid}> ${bodyParts[0]}`;
 						send.body = `${bodyParts.map((l) => `> ${l}`).join("\n")}\n\n${send.body}`;
+						const matrixReplyRegex = /^<mx-reply>.*<\/mx-reply>/gs
```

Comment:
This seems to be missing semicolumns....have you tried to run `npm run lint`?

Reply:
Oops. Thanks for the reminder. Added those semicolons in.